src/DependencyInjection/SurfnetStepupExtension.php

```diff
@@ -95,6 +95,11 @@ public function load(array $config, ContainerBuilder $container)
             $container->removeDefinition('surfnet_stepup.guzzle.gateway_api');
         }
 
+        $container->getDefinition('surfnet_stepup.locale_cookie_settings')
+            ->setArguments(
+                array_values($config['locale_cookie'])
```

Comment:
While this code is very short, I'd opt for an explicit array here. This way the CookieSettings constructor argument order isn't unexpectedly changed when changing the order of `locale_cookie`'s children. (Especially since it may *not break*, but simply swap values for e.g. `domain` and `path`.)